src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -266,6 +267,10 @@ public class SimpleEvents {
 				.description("Called when a player fishes something. This is not of much use yet.")
 				.examples("on fish:")
 				.since("1.0");
+		Skript.registerEvent("Arrow Pickup", SimpleEvent.class, PlayerPickupArrowEvent.class, "[player] pickup up [an] arrow")
```

Comment:
I prefer adding a syntax like the event's name
```suggestion
		Skript.registerEvent("Arrow Pickup", SimpleEvent.class, PlayerPickupArrowEvent.class, "[player] pickup [an] arrow", "[player] arrow pickup")
```

Reply:
This one is still not completed, please don't mark requested changes as completed unless they are.

Reply:
> I prefer adding a syntax like the event's name

It should be updated now 👍

Reply:
The new syntax part is still not resolved \:)
![image](https://user-images.githubusercontent.com/20037329/174070672-d2dd4578-37b5-4cd0-ad67-7226202afec3.png)

Reply:
> The new syntax part is still not resolved :) ![image](https://user-images.githubusercontent.com/20037329/174070672-d2dd4578-37b5-4cd0-ad67-7226202afec3.png)

Should be fixed now. Last time i try fixing an error in github editor, but it should be updated.